File: Python/Operadores/Intermedio/Ejercicio_05.py
# Área de un Cilindro
import math

radio = float(input("Ingrese el radio del cilindro: "))
altura = float(input("Ingrese la altura del cilindro: "))

area = 2 * math.pi * radio * (radio + altura)

print("El área del cilindro es:", area)

File: PHP/OPERADORES/Intermedio/Ejercicio_05.php
<?php
    $radio = 5;
    $altura = 10;

    //M_PI es una constante de PHP que guarda el valor de π
    $area = 2 * M_PI * $radio * ($radio + $altura);
    
    echo "El área del cilindro es:". round($area, 2);
?>